### PR TITLE
Second main

### PR DESCRIPTION
**Pull request by Mogle7Arkad**
please help review this task. Task not completed due to time constraint. Plan on working further on this task at a later time.

### CODE REVIEW

File: src/tabs.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        createFooter();
+    })
+
+    div3.addEventListener('click', () => {
+        clearContent();
+        createContactPage();
+        createFooter();
+    })
+    div4.addEventListener('click', () =>{
+
```
Comment (Phildeoner) [inline src/tabs.js:70]: This anonymous function is doing nothing why is it declared? Review it if it's useful it's advisable u take it out

File: src/restaurant.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    const contentDiv = document.createElement("div");
+    contentDiv.classList.add("content-div");
+
+    const container = document.createElement("div");
+    container.classList.add("container");
+    
+    const mainContent = document.createElement("div");
+    mainContent.classList.add("main-content");
+
```
Comment (Phildeoner) [inline src/restaurant.js:17]: Leave only one line space

**Review by Phildeoner — APPROVED** [review]
Hi @Mogle7Arkad 👋, you did a great job, well-done. How about you look into the recommendations, and also deploy the project on GitHub pages for easy access to the live preview. Your work is complete, the codes are well written, nothing more to say than to proceed to merge.
Happy coding 😀😀

File: src/tabs.js (hunk 1 above)
Comment (Mogle7Arkad) [reply src/tabs.js:70]: still have other eventlisteners to be added . the task is work in progress, have plans to do more actually.

File: src/restaurant.js (hunk 2 above)
Comment (Mogle7Arkad) [reply src/restaurant.js:17]: ok boss

**Pull request merged**